misc/metrics/examples/metrics_http_service.rs

```diff
@@ -0,0 +1,127 @@
+// Copyright 2022 Protocol Labs.
```

Comment:
`cargo` interprets this as a full example, how about doing the file structure below to have a single example only?

```
misc/metrics/examples/metrics/main.rs
misc/metrics/examples/metrics/http_service.rs
```